base/random.jl

```diff
@@ -1384,7 +1406,7 @@ function randexp_unlikely(rng, idx, x)
 end
 
 """
-    randn!([rng=GLOBAL_RNG], A::AbstractArray) -> A
+    randn!([rng::AbstractRNG=GLOBAL_RNG,] A::AbstractArray) -> A
```

Comment:
several of us would like to move towards separating signatures for leading optionals instead of continuing to use this non-syntax with square brackets, but that can be a task for later